uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/toil-request-ctrl.js

```diff
@@ -42,26 +42,29 @@ define([
 
       /**
        * Calculates toil expiry date.
-       * TODO It will be based on from date for both single and multiple days for now.
        *
        * @return {Promise}
        */
       vm.calculateToilExpiryDate = function () {
-        if (!vm.request.from_date) {
-          vm.errors = ['Please select from date to find expiry date'];
-
-          return $q.reject(vm.errors);
-        }
-
-        // If manager has already altered then it will directly show that date.
-        if (vm.request.toil_expiry_date) {
+        // blocks the expiry date from updating if not a manager and
+        // this is an open request.
+        if (!vm.canManage && vm.request.id) {
```

Comment:
This comment "this is an open request" and this condition "vm.request.id" don't quite match logically. May be the comment should be "this is an existing request"?